external/coreclr/coreclr.depproj

```diff
@@ -2,8 +2,6 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <OutputPath>$(RuntimePath)</OutputPath>
-     <!-- WORKAROUND: Force external packages to be restored for x64 until arm packages are fully broughtup-->
```

Comment:
I see the file was consolidated, but maybe we still want the comment in the new location (runtime.depproj)